### DISCUSSION TITLE
Control parameters

### DISCUSSION THREAD

**Discussion by z-master42** [reinhard-brandstaedter/solarflow-control#107] [Q&A]
Am I right?

min_charge_power = SolarFlow only starts to send something towards the house when the batteries are charged with at least the set power?

max_discharge_power = Maximum power draw from the battery?

max_inverter_limit = Should be clear.
                                                
limit_inverter = Throttle the inverter or not?

inverter_min_limit = Lower limit up to which or from which the inverter will work at all?

**Comment by reinhard-brandstaedter** [comment]
Most of that is correct, yes.
min_charge_power = can also be seen as reserverd charging power...the first X watts of SF's solarinput is used to charge the battery. the rest - if something is needed by the house demand - goes to the house. if nothing is needed everything is used to charge.

max_discharge_power = the maximum power that is output to home but only if needed. usually good enough to set this to your base load.

limit_inverter = is basically obsolete (leftover from prior versions). It must always be set True now, as solarflow-control now really makes use of limiting inverter AND solarflow.

inverter_min_limit = some inverters refuse to accept a limit below a certain value and then stop working/waking up. this value ensures solarflow control doesn't go below. 10 proved to be a good value for most inverters controlled by OpenDTU/AhoyDTU.

**Reply by z-master42** [reply]
Ok. This means that in future it will not (or no longer) be possible to regulate only the SolarFlow and leave the hub unlimited?
